src/redux/store.ts

```diff
@@ -8,10 +8,14 @@ import { SimulatorPlugin } from "../connection/sim";
 import { ConiqlPlugin } from "../connection/coniql";
 import { ConnectionForwarder } from "../connection/forwarder";
 
-const CONIQL_SOCKET = import.meta.env.VITE_CONIQL_SOCKET;
-const CONIQL_SSL = import.meta.env.VITE_CONIQL_SSL === "true";
+const CONIQL_SOCKET =
+  process.env.VITE_CONIQL_SOCKET ?? import.meta.env.VITE_CONIQL_SOCKET;
```

Comment:
I am having problems getting this to work with a client still running cra. I get the following error after issuing `npm run start`:
```
Failed to compile.

./node_modules/@diamondlightsource/cs-web-lib/dist/esm/index.js 19497:105
Module parse failed: Unexpected token (19497:105)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| }
| var _a$1, _b, _c, _d;
> const CONIQL_SOCKET = (_a$1 = process.env.VITE_CONIQL_SOCKET) !== null && _a$1 !== void 0 ? _a$1 : import.meta.env.VITE_CONIQL_SOCKET;
| const CONIQL_SSL = ((_b = process.env.VITE_CONIQL_SSL) !== null && _b !== void 0 ? _b : import.meta.env.VITE_CONIQL_SSL) === "true";
| const THROTTLE_PERIOD = parseFloat((_d = (_c = process.env.VITE_THROTTLE_PERIOD) !== null && _c !== void 0 ? _c : import.meta.env.VITE_THROTTLE_PERIOD) !== null && _d !== void 0 ? _d : "100");
```
I think this comes down to it not being able to parse `import.meta.env.*` ? Is this something you were able to workaround?

Reply:
I thought it had worked when I tested previously, but I just tried again now and encountered this same issue. I don't think there is a workaround for this.